providers/music/songkick.go

```diff
@@ -0,0 +1,89 @@
+## Feel free to ignore this file 
+package main
+
+import (
+	"errors"
+	"fmt"
+	"io/ioutil"
+
+	"net/http"
+)
+
+const songkickBaseAddr = "https://api.songkick.com/api/3.0"
```

Comment:
this should get thrown into the `Client` type as a default value, to allow for explicit proxying